### ISSUE TITLE
Download and fine-tuning and other question

### ISSUE THREAD

**Issue by xungeer29** [JiajiongCao/CELEB-500K#4]
Hello, who knows how to download these data quickly? I use the script in this repo to download images, but only about one million images downloaded a week, the total number of this dataset is 50M, so I need one year to download it, time is too long......

In addition, I use the downloaded images with 8000 IDs to clean the dataset as the described in the paper, but the top1 accuracy is 0.17 when I fine-tuning the pretrained model that trained in a clean dataset. The accuracy is too low to clean the dataset, did I make a mistake in fine-tuning?

**Comment by wuqiangch** [comment]
@xungeer29  I read your csdn. Have you done all images? I also download the images very  slowly.
